capstone-app/src/app/modules/photos/components/photos-section/photos-section.component.ts

```diff
@@ -34,26 +36,26 @@ export class PhotosSectionComponent implements OnInit, OnChanges {
   navItems: NavItem[] = [];
   markerFilter = '';
 
+  constructor(private readonly cd: ChangeDetectorRef, private places: SharedPlacesCityService) { }
+
   ngOnInit() {
     this.mockPlaces.set('Pjs Wings', ['Restuarant']);
     this.mockPlaces.set('Colonial Theater', ['History', 'Entertainment']);
     this.mockPlaces.set('Valley Forge', ['History', 'Landmark']);
+    combineLatest([this.places.getPlacesSource(), this.places.getCityName()]).subscribe(results => {
```

Comment:
optional: 
[...]).subscribe([placesSource, cityName] => {
      this.markerPlaces = placesSource;
      this.city = CityName;
      [...]

Reply:
Thanks, really like this